README.md

```diff
@@ -118,7 +145,6 @@ specification for a full list of information that can be retrieved.
     depstream_list = dep.get_runtime_streams('depstreamname')
     <do_stuff>
 ```
-
```

Comment:
Leave the newline here, please. The previous request was just to eliminate the unexpected whitespace you added.

Other than that, assuming the CI tests all pass, I think this is probably ready to merge.

Reply:
Ah, my bad. 2 minutes